1-org/modules/ml-org-policies/main.tf

```
@@ -0,0 +1,262 @@
+  allowed_vertex_vpc_networks = [for id in var.allowed_vertex_vpc_networks.parent_ids : format(local.under_format, id)]
+}
+
+module "ml_organization_policies_type_boolean" {
+  source   = "terraform-google-modules/org-policy/google"
+  version  = "~> 5.1"
+  for_each = local.ml_boolean_type_organization_policies
+
+  organization_id = var.org_id
+  folder_id       = var.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/${each.value}"
+}
+
+/******************************************
+  Cloud build
+*******************************************/
+
+module "allowed_integrations" {
+  #Allowed Integrations
+  #Control ID: CB-CO-6.2
+  #NIST 800-53: AC-3 AC-12 AC-17 AC-20
+  #CRI Profile: PR.AC-3.1 PR.AC-3.2 PR.AC-4.1 PR.AC-4.2 PR.AC-4.3 PR.AC-6.1 PR.AC-7.1 PR.AC-7.2 PR.PT-3.1 PR-PT-4.1
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.1"
+
+  organization_id   = var.org_id
+  folder_id         = var.folder_id
+  policy_for        = local.policy_for
+  policy_type       = "list"
+  allow_list_length = 1
+  allow             = var.allowed_integrations
+  constraint        = "constraints/cloudbuild.allowedIntegrations"
+}
+
+/*****************************************
+  Common
+******************************************/
+
+module "restrict_tls_versions" {
+  #Restrict TLS Versions Supported by Google APIs
+  #Control ID: COM-CO-1.1
+  #NIST 800-53: SC-8 SC-13
+  #CRI Profile: PR.DS-2.1 PR.DS-2.2 PR.DS-5.1
+
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.1"
+
+  organization_id  = var.org_id
+  folder_id        = var.folder_id
+  policy_for       = local.policy_for
+  policy_type      = "list"
+  deny_list_length = length(var.restricted_tls_versions)
+  deny             = var.restricted_tls_versions
+  constraint       = "constraints/gcp.restrictTLSVersion"
+}
+
+module "restrict_cmek_key_projects" {
+  #Customer Managed Encryption Keys (1 of 2)
+  #Control ID: COM-CO-2.2
+  #NIST 800-53 SC-12 SC-13
+  #CRI Profile: PR.DS-1.1 PR.DS-1.2 PR.DS-2.1 PR.DS-2.2 PR.DS-5.1
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.1"
+
+  organization_id   = var.org_id
+  folder_id         = var.folder_id
+  policy_for        = local.policy_for
+  policy_type       = "list"
+  allow_list_length = 1
+  allow             = local.access_scope
+  constraint        = "constraints/gcp.restrictCmekCryptoKeyProjects"
+}
+
+module "restrict_non_cmek_services" {
+  #Customer Managed Encryption Keys (2 of 2)
+  #Control ID: COM-CO-2.3
+  #NIST 800-53: SC-12
+  #CRI Profile: PR.DS-1.1 PR.DS-1.2 PR.DS-2.1 PR.DS-2.2 PR.DS-5.1
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.1"
+
+  organization_id  = var.org_id
+  folder_id        = var.folder_id
+  policy_for       = local.policy_for
+  policy_type      = "list"
+  deny_list_length = length(var.restricted_tls_versions)
```

I think maybe here the deny list length is misleading

```suggestion
  deny_list_length = length(var.restricted_non_cmek_services)
```